lib/bug_report_templates/rails_5_master.rb

```diff
@@ -58,7 +54,11 @@ class TestApp < Rails::Application
   Rails.logger  = config.logger
 end
 
-class YourModel < ActiveRecord::Base
+class ApplicationController < ActionController::Base
+  include Rails.application.routes.url_helpers
```

Comment:
I guess this change belongs in the previous commit... :) I'm a git history freak, I know.